tensorflow/core/kernels/crop_resize_bilinear_core.h

```diff
@@ -0,0 +1,5582 @@
+  pack4_2b_1ch_(v0, v1, v2, v3);
+}
+template <>
+void VectorLoader<float>::pack_1ch(__m128i* v0, __m128i* v1, __m128i* v2,
+                                   __m128i* v3) {
+  pack4_4b_1ch_(v0, v1, v2, v3);
+}
+
+template <>
+void VectorLoader<uint8>::pack_2ch(__m128i* v0, __m128i* v1, __m128i* v2,
+                                   __m128i* v3) {
+  pack4_1b_2ch_(v0, v1, v2, v3);
+}
+template <>
+void VectorLoader<int8>::pack_2ch(__m128i* v0, __m128i* v1, __m128i* v2,
+                                  __m128i* v3) {
+  pack4_1b_2ch_(v0, v1, v2, v3);
+}
+template <>
+void VectorLoader<uint16>::pack_2ch(__m128i* v0, __m128i* v1, __m128i* v2,
+                                    __m128i* v3) {
+  pack4_2b_2ch_(v0, v1, v2, v3);
+}
+template <>
+void VectorLoader<int16>::pack_2ch(__m128i* v0, __m128i* v1, __m128i* v2,
+                                   __m128i* v3) {
+  pack4_2b_2ch_(v0, v1, v2, v3);
+}
+template <>
+void VectorLoader<int32>::pack_2ch(__m128i* v0, __m128i* v1, __m128i* v2,
+                                   __m128i* v3) {
+  pack4_4b_2ch_(v0, v1, v2, v3);
+}
+template <>
+void VectorLoader<Eigen::half>::pack_2ch(__m128i* v0, __m128i* v1, __m128i* v2,
+                                         __m128i* v3) {
+  pack4_2b_2ch_(v0, v1, v2, v3);
+}
+template <>
+void VectorLoader<bfloat16>::pack_2ch(__m128i* v0, __m128i* v1, __m128i* v2,
+                                      __m128i* v3) {
+  pack4_2b_2ch_(v0, v1, v2, v3);
+}
+template <>
+void VectorLoader<float>::pack_2ch(__m128i* v0, __m128i* v1, __m128i* v2,
+                                   __m128i* v3) {
+  pack4_4b_2ch_(v0, v1, v2, v3);
+}
+
+template <>
+void VectorLoader<uint8>::pack_3ch(__m128i* v0, __m128i* v1, __m128i* v2,
+                                   __m128i* v3) {
+  pack4_1b_3ch_(v0, v1, v2, v3);
+}
+template <>
+void VectorLoader<int8>::pack_3ch(__m128i* v0, __m128i* v1, __m128i* v2,
+                                  __m128i* v3) {
+  pack4_1b_3ch_(v0, v1, v2, v3);
+}
+template <>
+void VectorLoader<uint16>::pack_3ch(__m128i* v0, __m128i* v1, __m128i* v2,
+                                    __m128i* v3) {
+  pack4_2b_3ch_(v0, v1, v2, v3);
+}
+template <>
+void VectorLoader<int16>::pack_3ch(__m128i* v0, __m128i* v1, __m128i* v2,
+                                   __m128i* v3) {
+  pack4_2b_3ch_(v0, v1, v2, v3);
+}
+template <>
+void VectorLoader<int32>::pack_3ch(__m128i* v0, __m128i* v1, __m128i* v2,
+                                   __m128i* v3) {
+  pack4_4b_3ch_(v0, v1, v2, v3);
+}
+template <>
+void VectorLoader<Eigen::half>::pack_3ch(__m128i* v0, __m128i* v1, __m128i* v2,
+                                         __m128i* v3) {
+  pack4_2b_3ch_(v0, v1, v2, v3);
+}
+template <>
+void VectorLoader<bfloat16>::pack_3ch(__m128i* v0, __m128i* v1, __m128i* v2,
+                                      __m128i* v3) {
+  pack4_2b_3ch_(v0, v1, v2, v3);
+}
+template <>
+void VectorLoader<float>::pack_3ch(__m128i* v0, __m128i* v1, __m128i* v2,
+     
```

Comment:
This assertion is failing in one of the tests:

F1002 18:47:12.186167    5518 logging.cc:80] assert.h assertion failed at ./third_party/tensorflow/core/kernels/crop_resize_bilinear_core.h:3621 in tensorflow::(anonymous namespace)::CropResizeCastImage<float, float>::CropResizeCastImage(const int, const int, const int, const int, const int, const int, const int, const tensorflow::(anonymous namespace)::CachedInterpolation *, const int, const int, const tensorflow::(anonymous namespace)::CachedInterpolation *, const float, const bool, const bool, const bool, const int) [T = float, U = float]: min_ix_ <= max_ix_
*** Check failure stack trace: ***
    @     0x559e7befccd2  base_logging::LogMessage::SendToLog()
    @     0x559e7befd4d3  base_logging::LogMessage::Flush()
    @     0x559e7beff3d5  base_logging::LogMessageFatal::~LogMessageFatal()
    @     0x559e7befa590  __assert_fail
    @     0x559e72986c68  tensorflow::(anonymous namespace)::CropResizeCastImage<>::CropResizeCastImage()
    @     0x559e729868f0  tensorflow::(anonymous namespace)::crop_resize_single_image_common<>()
    @     0x559e729b99eb  tensorflow::functor::CropAndResize<>::operator()()::{lambda()#1}::operator()()
    @     0x559e729b947e  std::_Function_handler<>::_M_invoke()
    @     0x559e7747d502  std::function<>::operator()()
    @     0x559e7a064168  tensorflow::thread::ThreadPool::Impl::ParallelFor()::{lambda()#1}::operator()()
    @     0x559e7a063fda  std::_Function_handler<>::_M_invoke()
    @     0x559e709b87f2  std::function<>::operator()()
    @     0x559e709b7149  Eigen::ThreadPoolDevice::parallelFor()
    @     0x559e709b80a5  Eigen::ThreadPoolDevice::parallelFor()
    @     0x559e7a05bfc7  tensorflow::thread::ThreadPool::Impl::ParallelFor()
    @     0x559e7a05acef  tensorflow::thread::ThreadPool::ParallelFor()
    @     0x559e79ee3380  tensorflow::Shard()
    @     0x559e729b9314  tensorflow::functor::CropAndResize<>::operator()()
    @     0x559e729b8eb9  tensorflow::CropAndResizeOp<>::ComputeAsync()::{lambda()#1}::operator()()
    @     0x559e729b8bdd  std::_Function_handler<>::_M_invoke()
    @     0x559e6e44ba3d  std::function<>::operator()()
    @     0x559e72911ab3  tensorflow::(anonymous namespace)::RunIfBoxIndexIsValid<>()
    @     0x559e729b8ab9  tensorflow::CropAndResizeOp<>::ComputeAsync()
    @     0x559e76c285ed  tensorflow::Device::ComputeAsync()
    @     0x559e79c168b6  tensorflow::(anonymous namespace)::ExecutorState::Process()
    @     0x559e79c17395  tensorflow::(anonymous namespace)::ExecutorState::ScheduleReady()::$_1::operator()()
    @     0x559e79c171bd  std::_Function_handler<>::_M_invoke()
    @     0x559e6e44ba3d  std::function<>::operator()()
    @     0x559e7a05ed85  tensorflow::thread::EigenEnvironment::ExecuteTask()
    @     0x559e7a05e0d9  Eigen::ThreadPoolTempl<>::WorkerLoop()
    @     0x559e7a05dd1e  Eigen::ThreadPoolTempl<>::ThreadPoolTempl()::{lambda()#1}::operator()()
    @     0x559e7a05db8d  std::_Function_handler<>::_M_invoke()
*